pkgs/servers/http/apache-modules/mod_ca/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, gnused, coreutils, pkgconfig, apacheHttpd, openssl, openldap }:
+
+stdenv.mkDerivation rec {
+ baseurl = "https://redwax.eu/dist/rs/";
+ name = "mod_ca";
+ suffix = ".tar.gz";
+
+ meta = with stdenv.lib; {
```

Comment:
Typically we put this at the end of the file. Not a strict requirement or anything... but everyone else is doing it :sunglasses:

Reply:
Will fix :) copied it from the wrong example I guess :) : )

Reply:
fixed.